app/src/main/res/layout/fragment_year_picker.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
@manuelplazaspalacio Can you refactor year dialog view style according to look and feel of the app?